server.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const morgan = require('morgan');
+
+const server = express();
+
+server.get('/', (req, res) => {
+	res.send('SERVER IS WORKING');
```

Comment:
Starting up great! When you've got more routes up, I'll come back for a more thorough review. :) Let me know if you need any help!